src/index.js

```diff
@@ -12,23 +12,36 @@ path = require('path'),
 js = require('escomplex-js'),
 coffee = require('escomplex-coffee'),
 escomplex = require('escomplex'),
-check = require('check-types');
+check = require('check-types'),
+async = require('async');
+
 
 parseCommandLine();
 
 state = {
-    starting: true,
-    openFileCount: 0,
     sources: {
-      js: [],
-      coffee: []
+        js: [],
+        coffee: []
     },
-    tooComplex: false,
-    failingModules: []
 };
 
 expectFiles(cli.args, cli.help.bind(cli));
-readFiles(cli.args);
+queue = async.queue(function(filePath, cb) {
```

Comment:
I think I'd prefer it if this inner function was named and pulled down to sit with the rest of the logic lower down the file.